examples/docker/census-on-omnisci/census-omnisci.dockerfile

```diff
@@ -46,8 +46,9 @@ RUN conda update -n base -c defaults conda -y && \
     conda config --set channel_priority strict && \
     conda create -n modin --yes --no-default-packages && \
     conda activate modin && \
-    conda install -c intel/label/modin -c conda-forge -c intel \
-        modin "ray>=1.0.0" "numpy==1.20.3" "scikit-learn<0.24.0" "daal4py>=2021.1" dpcpp_cpp_rt && \
+    conda install -c intel/label/modintest -c conda-forge \
```

Comment:
what is the reason for using omnisci from `intel/label/modintest` (the name implies that the bits might not be stable) instead of taking `pyomniscidbe` from conda-forge?

Reply:
I did not consider `pyomniscidbe` since Modin has not yet switched to using it. Do I have to wait for this moment?

`intel/label/modintest` channel contains the most current version, I think it would be interesting to use it.

Reply:
Using modin from conda-forge now.